install-jdk.sh

```diff
@@ -248,7 +248,7 @@ function download_and_extract_and_set_target() {
     local local="--directory-prefix ${workspace}"
     local remote='--timestamping --continue'
     local wget_options="${quiet} ${local} ${remote}"
-    local tar_options="-z --file ${archive}" # "--auto-compress" is not supported on mac osx, using "-z"
+    local tar_options="--file ${archive}" # "--auto-compress" is not supported on mac osx, using "-z"
```

Comment:
The comment is now out-of-sync with the code, no?

Reply:
Aye. Removing it.

Reply:
Sorry, should have done this but I did not understand it - now I do ;)